### PR TITLE
RABSW-1081: Support multiple MDTs

### PR DESCRIPTION
**Pull request by matthew-richerson**
Signed-off-by: Matt Richerson <matthew.richerson@Matts-MacBook-Pro.local>

### CODE REVIEW

**Review by roehrich-hpe — APPROVED** [review]

**Review by NateThornton — APPROVED** [review]
Is there a need for new/additional unit tests?

**Comment by matthew-richerson** [comment]
> 

Yes, you and Dean are early to the party, though. This is just a draft PR so I can get an image built to test on htx-2.

**Comment by matthew-richerson** [comment]
This PR is ready for review now.

**Review by NateThornton — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/HewlettPackard/dws v0.0.0-20230104194155-1f78117736f5
 	github.com/NearNodeFlash/lustre-fs-operator v0.0.0-20220929204230-5dcfe552c9e0
-	github.com/NearNodeFlash/nnf-ec v0.0.0-20221221152437-83d47b38ec80
+	github.com/NearNodeFlash/nnf-ec v0.0.0-20230105221007-3282bfd76a70
```
Comment (NateThornton) [inline go.mod:8]: Just a reminder to update this once nnf-ec master is ready

**Review by NateThornton — APPROVED** [review]

**Pull request merged**